packages/embed/src/js/smartselfie-auth.js

```diff
@@ -57,6 +56,12 @@ import { version as sdkVersion } from '../../package.json';
         event.data.includes('SmileIdentity::Configuration')
       ) {
         config = JSON.parse(event.data);
+        if (config.use_new_component) {
```

Comment:
should this config variable be scoped to the product it is applicable to?

Reply:
If I get you right, both smart selfie auth and smart selfie camera will use the new component for selfie capture.
Right now only products that require selfie and document capture make use of the new component